.github/workflows/upload-translations.yml

```diff
@@ -0,0 +1,36 @@
+name: add to codecov [v6]
+
+on:
+  push:
+    branches-ignore:
+      - 'gh-pages'
+      - 'gh-pages-history'
+      - 'master'
+      - 'develop'
+    tags-ignore:
+      - '*'
+    paths:
+      - './src/translations/**.json'
+  pull_request:
+    branches:
+      - develop
+    paths:
+      - './src/translations/**.json'
+
+jobs:
+  upload-translaations:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check and create lokalise branch
+        env:
+          TOKEN: ${{secrets.LOKALISE_TOKEN}}
+          PROJECT_ID: ${{secrets.PROJECT_ID}}
+        run: TOKEN=TOKEN BRANCH_NAME=${{github.ref}} PROJECT_ID=PROJECT_ID node ./scripts/createLokaliseBranch.js
```

Comment:
it should be `TOKEN=$TOKEN BRANCH_NAME=$GITHUB_REF PROJECT_ID=$PROJECT_ID node ./scripts/createLokaliseBranch.js`

Reply:
actually you dont have to pass the TOKEN, PROJECT_ID since you are reading the process.env at the script, you can change the script to get process.env.GITHUB_REF to get the branch name